packages/off-chain/inputs/Contributions.json

```diff
@@ -0,0 +1,42 @@
+[
```

Comment:
@Zitzak This is how the IE function is currently expecting the input to look like, fyi